homeassistant/components/senz/manifest.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "senz",
+  "name": "nVent RAYCHEM SENZ",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/senz",
+  "requirements": ["aiosenz==1.0.0"],
+  "dependencies": ["http"],
```

We recently changed which integration is responsible for the oauth2 endpoint.
```suggestion
  "dependencies": ["auth"],
```